vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go

```diff
@@ -254,10 +247,17 @@ func getSELinuxPolicyRoot() string {
 	return filepath.Join(selinuxDir, readConfig(selinuxTypeTag))
 }
 
-func isProcHandle(fh *os.File) (bool, error) {
+func isProcHandle(fh *os.File) error {
 	var buf unix.Statfs_t
 	err := unix.Fstatfs(int(fh.Fd()), &buf)
-	return buf.Type == unix.PROC_SUPER_MAGIC, err
+	if err != nil {
+		return fmt.Errorf("statfs(%q) failed: %v", fh.Name(), err)
+	}
+	if buf.Type != unix.PROC_SUPER_MAGIC {
```

Comment:
@kolyshkin I think this should also be treated the same?

```suggestion
	if uint32(buf.Type) != uint32(unix.PROC_SUPER_MAGIC) {
```

Reply:
oh, nevermind; looks like this `const` is not problematic